docusaurus.config.js

```diff
@@ -46,7 +46,7 @@ const config = {
         docs: {
           routeBasePath: "/",
           sidebarPath: require.resolve("./sidebars.js"),
-          editUrl: "https://github.com/trypear/pearai-app/tree/main/docs",
+          editUrl: "https://github.com/trypear/pearai-documentation/blob/main/",
```

Comment:
I noticed you also changed the url here so will you close the other pr?

Reply:
https://github.com/trypear/pearai-documentation/pull/11 this specific pr

Reply:
Yeah, it's closed now